api/namex/models/request.py

```diff
@@ -114,7 +114,7 @@ class Request(db.Model):
     checkedOutDt = db.Column('checked_out_dt', db.DateTime(timezone=True),index=True)
 
     # MRAS fields
-    homeJurisNum = db.Column('home_juris_num', db.String(40))
+    homeJurisNum = db.Column('home_juris_num', db.String(150))
```

Comment:
Hey wait a minute. I saw a Namerequest UI PR recently that changed this from 150 to 40. Which is correct???

https://github.com/bcgov/namerequest/commit/ed8d6d2e131ac241a4a4f2359849688c6dfbd8ae